src/http_fork_source.rs

```diff
@@ -0,0 +1,65 @@
+use eyre::Context;
+use zksync_web3_decl::{
+    jsonrpsee::http_client::{HttpClient, HttpClientBuilder},
+    namespaces::{EthNamespaceClient, ZksNamespaceClient},
+};
+
+use crate::fork::{block_on, ForkSource};
+
+#[derive(Debug)]
+/// Fork source that gets the data via HTTP requests.
+pub struct HttpForkSource {
+    /// URL for the network to fork.
+    pub fork_url: String,
+}
+
+impl HttpForkSource {
+    pub fn create_client(&self) -> HttpClient {
+        HttpClientBuilder::default()
+            .build(self.fork_url.clone())
+            .expect(&format!(
```

Comment:
lint: This might be a new rule with the latest clippy, but looks like it's expecting this change

```suggestion
             .unwrap_or_else(|_| panic!("Unable to create a client for fork: {}",
                 self.fork_url))
```